src/screens/repo/screens/builds/index.less

```diff
@@ -1,3 +1,24 @@
+@import "~shared/styles/colors";
+
 .root {
-	padding: 20px;
+  padding: 20px;
+}
+
+button {
+  background: @white;
+  border: 1px solid @gray-dark;
```

Comment:
I'll suggest to use `@gray-light` instead of `@gray-dary` to use the same convention we have on other boxed text

Reply:
I think I borrowed the styling from some forms to try and keep things consistent, either from [here]( https://github.com/drone/drone-ui/blob/master/src/screens/repo/screens/registry/components/form.less#L39) or [here]( https://github.com/drone/drone-ui/blob/master/src/screens/repo/screens/secrets/components/form.less#L39).

I do see the action/log buttons have `@gray-light` or `@gray` but they contain SVGs (see [here](https://github.com/drone/drone-ui/blob/master/src/screens/repo/screens/build/logs/index.less#L30) and [here](https://github.com/drone/drone-ui/blob/master/src/screens/repo/screens/build/logs/index.less#L71)).

Reply:
@wass3r actually we have `@gray-dark` border on button that indicate form submit (eg: secret settings for a repo) and `@gray-light` on boxed element like "follow log" button in build page.
IMHO this element have the same "meaning" that "follow build log" have, so I prefer light border, but I not stronger opinion about it. Thanks for investigate it 👍